src/main/features/dashboard/views/defendant.njk

```diff
@@ -39,7 +39,7 @@
           </li>
           <li>
             <strong class="bold">{{ t('Claim details:') }}</strong>
-            <span class="block"><a href="{{ DashboardPaths.claimReceiptReceiver.evaluateUri({ externalId: claim.externalId }) }}">{{ t('Download claim') }}</a></span>
+            <span class="block"><a href="{{ ResponsePaths.claimDetailsPage.evaluateUri({ externalId: claim.externalId }) }}">{{ t('View claim') }}</a></span>
```

Comment:
When response already exists and defendant clicks the link then an application is doing redirect to dashboard page with the following message in logs:

```
2018-04-04T01:28:53+01:00 WARN response/guards/responseGuard: State guard: response already exists - redirecting to dashboard
```

Please make it work in all cases.